core/src/test/java/org/apache/calcite/plan/RelWriterTest.java

```diff
@@ -675,15 +661,57 @@
     RelJsonWriter jsonWriter = new RelJsonWriter();
     relNode.explain(jsonWriter);
     final String relJson = jsonWriter.asString();
-    String s = deserializeAndDumpToTextFormat(getSchema(relNode), relJson);
+    String s = deserializeAndDumpToTextFormat(getSchema(relNode), relJson, SqlExplainFormat.TEXT);
```

Comment:
The method name already tells us that it is dumping to TEXT format...

Reply:
oh, I miss that. Thanks.